src/collections/_documentation/error-reporting/configuration/drain-example/javascript.md

```diff
@@ -1,12 +1,15 @@
-The client provides a close method that optionally takes the time in milliseconds for how long it
-waits and will return a promise that resolves when everything was flushed or the timeout kicked
+The `close` method optionally takes a timeout in milliseconds and returns a
+promise that resolves when all pending events were flushed or the timeout kicked
 in.
 
 ```javascript
-const client = Sentry.getCurrentHub().getClient();
-if (client) {
-  client.close(2000).then(function() {
-    process.exit();
-  });
-}
+Sentry.close(2000).then(function() {
+  process.exit();
+});
```

Comment:
@HazAT @kamilogorek was there a reason to manually get the client instead of using `Sentry.close` that does the same?

Reply:
Not really, you are right it's the same.